### PR TITLE
Scripts for zowe launcher

### PR DESCRIPTION
**Pull request by lchudinov**
#### PR type
- [x] Feature 

#### Relevant issues
https://app.zenhub.com/workspaces/community-5c93e02fa70b456d35b8f0ed/issues/zowe/zlux/590

#### Changes proposed in this PR
- PR adds scripts for Zowe Launcher

#### Does this PR introduce a breaking change?

- [x] No

### CODE REVIEW

File: bin/internal/run-zowe.sh [hunk 1]
```diff
@@ -36,7 +36,11 @@ export ROOT_DIR=$(cd $(dirname $0)/../../;pwd)
 launch_components_list=$(${ROOT_DIR}/bin/internal/get-launch-components.sh -c "${INSTANCE_DIR}" -r "${ROOT_DIR}")
 . ${ROOT_DIR}/bin/internal/prepare-workspace.sh -c "${INSTANCE_DIR}" -r "${ROOT_DIR}" -t "${launch_components_list}"
 
-# FIXME: Zowe Launcher can take responsibility from here
-for component_id in $(echo "${launch_components_list}" | sed "s/,/ /g"); do
-  . ${ROOT_DIR}/bin/internal/start-component.sh -c "${INSTANCE_DIR}" -r "${ROOT_DIR}" -o "${component_id}" &
-done
+if [ "${USE_LAUNCHER}" = "true" ]
+then
+  ${ROOT_DIR}/components/launcher/bin/zowe_launcher "${launch_components_list}"
```
Comment (1000TurquoisePogs) [inline bin/internal/run-zowe.sh:41]: The launcher isn't a component but it starts components. There should be a different location for zowe launcher. Perhaps ${ROOT_DIR}/bin/launcher or similar.

**Review by 1000TurquoisePogs — CHANGES_REQUESTED** [review]
Zowe launcher path needs to be changed somehow.
https://github.com/zowe/launcher/pull/18 mentions /bin/zowe_launcher
That could be OK, but lets hear from Jack.

**Comment by jackjia-ibm** [comment]
I agree `${ROOT_DIR}/bin/launcher` is a better location. Launcher is packaged with manifest, but it's not a regular component. It makes sense to put into `${ROOT_DIR}/bin` folder to avoid confusion.

**Comment by jackjia-ibm** [comment]
Hi @lchudinov, I'm thinking to merge this PR with YAML config related changes in https://github.com/zowe/zowe-install-packaging/pull/2049. To make it compatible, I made a small change on the shell script name here https://github.com/zowe/launcher/pull/18/commits/73541a9de4c6bb2fa7d2c5d72399dbf74f642f37#diff-e0cf5b28d9b6b600f0af2bc78e8fd30ec675fd731a5da86f0c4283ffc0e40176L852-R879.

I will do an integration test first and make changes related to process IDs handling. Later I will need your help to make a change to add HA_INSTANCE_ID as procedure parameter. It will be a pass through parameter. Thanks a lot Leonty!

**Comment by jackjia-ibm** [comment]
Close because similar features had been merged into #2049 and merged into staging branch.

**Pull request closed without merging**